rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.adoc

```diff
@@ -8,13 +8,23 @@ toc::[]
 
 Create a {product-title} (ROSA) cluster quickly by using the default options and automatic AWS Identity and Access Management (IAM) resource creation. You can deploy your cluster by using {cluster-manager-first} or the ROSA CLI (`rosa`).
 
-The procedures in this document use `auto` mode to immediately create the required IAM resources using the current AWS account, including the account-wide IAM roles and policies, Operator policies, cluster-specific Operator roles, and the OpenID Connect (OIDC) identity provider.
+The procedures in this document use `auto` mode to immediately create the required IAM resources using the current AWS account. The required resources include the account-wide IAM roles and policies, Operator policies, cluster-specific Operator roles, and OpenID Connect (OIDC) identity provider.
```

Comment:
The account-wide IAM roles: Installer, Support, Worker, and Control plane are created in advance using the CLI, and not as part of the `auto` mode flow. Maybe we should mention that?

Reply:
@oriAdler hi! Thank you for your review and for this suggestion.

In the http://file.fab.redhat.com/pneedle/pr49293/rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.html page, the steps to create the account-wide roles are also included. I guess what is confusing is that there is an `auto` mode in the ROSA CLI and an `auto` mode in the OCM workflow.

In regard to creating the account-wide roles, both of the following sections on the page use auto mode to create the account-wide roles:

- In http://file.fab.redhat.com/pneedle/pr49293/rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.html, the `rosa create account-roles` command defaults to using `auto` mode within the ROSA CLI, to create the roles
- In http://file.fab.redhat.com/pneedle/pr49293/rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.html#rosa-sts-creating-a-cluster-quickly-cli_rosa-sts-creating-a-cluster-quickly, the `rosa create account-roles --mode auto` command is passed. 

I will have a think about how to update the introductory line to be more explicit.

Reply:
@oriAdler I have updated this line to be more explicit about the use of the automatic modes in the ROSA CLI and OpenShift Cluster Manager. Can you please let me know if this resolves your suggestion? Thanks!

Reply:
Thanks for the clarification! SGTM.

Reply:
Thank you!